### PR TITLE
second PR deep0713

### PR DESCRIPTION
**Pull request by deep0713**
**2nd pull  request**

### CODE REVIEW

**Review by krishnadevz — APPROVED** [review]
he/she added himself in teams I am approving this pull

File: firstpr/deepak1_try2.md [hunk 1]
```diff
@@ -1 +1,3 @@
-Deepak- i have no experience in coding except C at college level and HTML & JS as hobby ( Self taught).
+Deepak- i have no experience in coding except C at college level and HTML & JS as hobby ( Self taught). 
+
+making chamges
```
Comment (tr0mbl3y) [inline firstpr/deepak1_try2.md:3]: you can remove these lines

Comment (krishnadevz) [reply firstpr/deepak1_try2.md:3]: you can remove those lines

**Review by krishnadevz — COMMENTED** [review]
you can remove those lines

File: firstpr/deepak1_try2.md (hunk 1 above)
Comment (deep0713) [reply firstpr/deepak1_try2.md]: I have made the the changes you asked.

**Review by ni3-kdm — APPROVED** [review]

**Review by shekhar10feb — APPROVED** [review]

**Pull request merged**